### PR TITLE
WA-72: Added basic tracking

### PR DESCRIPTION
**Pull request by rmeissner**
Closes WA-72.

Changes proposed in this pull request:
- Added animations to unlock screen
- Added basic tracking

@gnosis/mobile-devs

### CODE REVIEW

File: app/src/main/java/pm/gnosis/heimdall/reporting/EventTracker.kt [hunk 1]
```diff
@@ -4,9 +4,56 @@ import io.reactivex.Single
...
+    SAFE_DETAILS_CREATE_TRANSACTION,
+    SAFE_OVERVIEW_SCAN_TRANSACTION,
+}
+
+enum class TabId {
+    ADD_NEW_SAFE,
+    ADD_SAFE_EXISTING,
+    SAFE_DETAILS_ASSETS,
+    SAFE_DETAILS_TRANSACTIONS,
+    SAFE_DETAILS_SETTINGS,
```
Comment (fmrsabino) [inline app/src/main/java/pm/gnosis/heimdall/reporting/EventTracker.kt]: Why tabs and not screens?

File: app/src/main/java/pm/gnosis/heimdall/reporting/impl/FabricEventTracker.kt [hunk 2]
```diff
@@ -1,19 +1,55 @@
 package pm.gnosis.heimdall.reporting.impl
 
+import com.crashlytics.android.Crashlytics
```
Comment (fmrsabino) [inline app/src/main/java/pm/gnosis/heimdall/reporting/impl/FabricEventTracker.kt]: Unecessary

File: app/src/main/java/pm/gnosis/heimdall/reporting/impl/FabricEventTracker.kt [hunk 3]
```diff
@@ -1,19 +1,55 @@
...
+import timber.log.Timber
 import javax.inject.Inject
 import javax.inject.Singleton
 
 @Singleton
 class FabricEventTracker @Inject constructor(
         private val fabricCore: FabricCore
 ) : EventTracker {
+
+    private val answers = Answers.getInstance()
```
Comment (fmrsabino) [inline app/src/main/java/pm/gnosis/heimdall/reporting/impl/FabricEventTracker.kt]: Can this be provided with Dagger?

Comment (rmeissner) [reply app/src/main/java/pm/gnosis/heimdall/reporting/impl/FabricEventTracker.kt]: I wouldn't, I could probably access it via the fabric core ... but it's the same as with the Crashlytics singleton

File: app/src/main/java/pm/gnosis/heimdall/reporting/EventTracker.kt (hunk 1 above)
Comment (rmeissner) [reply app/src/main/java/pm/gnosis/heimdall/reporting/EventTracker.kt]: because they are on the same screen, as different tabs

Comment (fmrsabino) [reply app/src/main/java/pm/gnosis/heimdall/reporting/EventTracker.kt]: I also see them as screens 😉

File: app/src/main/java/pm/gnosis/heimdall/reporting/impl/FabricEventTracker.kt (hunk 3 above)
Comment (fmrsabino) [reply app/src/main/java/pm/gnosis/heimdall/reporting/impl/FabricEventTracker.kt]: Yea. It's just to try to do more DI.

**Pull request merged**